src/d3a/models/strategy/update_frequency.py

```diff
@@ -159,6 +165,7 @@ def _post_bids(self, market, strategy):
                               bid.energy, buyer_origin=bid.buyer_origin)
 
     def update_posted_bids_over_ticks(self, market, strategy):
```

Comment:
To me at least, the existence of both bid-related and offer-related things in this class (that get differentiated only by method name) looks quite error-prone. How about creating 2 subclasses of this (one for the offers, one for the bids), in order to separate these functions ?